store_test.go

```diff
@@ -371,7 +372,12 @@ func TestLimitStore(t *testing.T) {
 	traces, _ := ms.Traces()
 	want := []*Trace{
 		{Span: Span{ID: SpanID{2, 3, 4}}},
-		{Span: Span{ID: SpanID{3, 4, 5}}},
+		{
+			Span: Span{ID: SpanID{3, 5, 6}},
+			Sub: []*Trace{
+				{Span: Span{ID: SpanID{3, 4, 5}}},
```

Comment:
It confused me initially as well, but I've confirmed it's valid. Those `SpanID` parameters are in order of `Trace`, `Span`, and `Parent`. That is span `3, 4, 5` say's it's parent is `Span` `5`  (i.e. `3, 5, 6`).